### ISSUE TITLE
Unable to resolve host 

### ISSUE THREAD

**Issue by exedox** [jacklul/pihole-updatelists#100]
So about a month ago I started having this issue where the page looks off:

![image](https://user-images.githubusercontent.com/55308770/155487281-080d79f4-fc68-4c6c-a221-a8dece763636.png)

So I checked the error log and came to find that there's an issue with resolving the host.  I'm not sure how to fix it.  I'm using a synology docker.  It was working in January though.  Is there something wrong on my end?  I used the same set up as in the instructions, with my own NAS ip as host.

<html><body>

2022-02-24 08:20:53 | stdout | sudo: unable to resolve host jacklul-pihole: Name or service not known
-- | -- | --
2022-02-24 08:20:52 | stdout | sudo: unable to resolve host jacklul-pihole: Name or service not known
2022-02-24 08:20:52 | stdout | sudo: unable to resolve host jacklul-pihole: Name or service not known
2022-02-24 08:20:41 | stdout | [services.d] done.
2022-02-24 08:20:41 | stdout | Starting crond
2022-02-24 08:20:41 | stdout | Starting pihole-FTL (no-daemon) as root
2022-02-24 08:20:41 | stdout | Starting lighttpd
2022-02-24 08:20:41 | stdout | [services.d] starting services
2022-02-24 08:20:41 | stdout | [cont-init.d] done.
2022-02-24 08:20:41 | stdout | [cont-init.d] 30-pihole-updatelists.sh: exited 0.

</body>
</html>

**Comment by jacklul** [comment]
I honestly have no idea where does that error comes from, is your container named 'jacklul-pihole' by any chance?

**Comment by exedox** [comment]
> 

OH wow you are so quick to respond! Thanks for that!  Yea it is the jacklul container.  Also the log is happening in reverse time (not sure why it exported that way).  I found out it looked odd because I was using the url http://address:port/admin/index.php/index.php#

It went back to normal when I did just http://address:port, so it was definitely my bad.

Thanks so much for creating this container and updatelist!  It's been super nice in auto-updating my pihole. I'm super grateful for it!
